common/constant/key.go

```diff
@@ -97,6 +97,11 @@ const (
 	ROLE_KEY             = "registry.role"
 	REGISTRY_DEFAULT_KEY = "registry.default"
 	REGISTRY_TIMEOUT_KEY = "registry.timeout"
+	REGISTRY_LABEL_KEY   = "label"
+	PREFERRED_KEY        = "preferred"
+	ZONE_KEY             = "zone"
+	REGISTRY_ZONE        = "registry_zone"
+	REGISTRY_ZONE_FORCE  = "registry_zone_force"
```

Comment:
I can‘t find where do you set this `REGISTRY_ZONE_FORCE` key?

Reply:
Beacuse this tag is from consumer’s attachments. Need consumer specify by self. I only run this in testing TestZoneWareInvokerWithZoneForceFail.

Reply:
I think may be add ClusterInterceptor is needed.